pkg/promql/fuzz.go

```diff
@@ -57,7 +57,13 @@ const (
 )
 
 func fuzzParseMetricWithContentType(in []byte, contentType string) int {
-	p := textparse.New(in, contentType)
+	p, warning := textparse.New(in, contentType)
+	if warning != nil {
+		// An invalid content type is being passed, which should not happen
+		// in this context.
+		panic(warning)
```

Comment:
Should we really panic here because of invalid contentType? I am not sure where this is being used

Reply:
This is a direct copy from Prometheus 

https://github.com/prometheus/prometheus/blob/957092451175e9c4b37c52b6d3c8444c1d3eebc4/promql/fuzz.go#L64

But I agree, we can replace that with a warning log.

Reply:
Yeah I've noticed it's borrowed code. Just wanted to make sure we have full understanding and doing the right thing in our context.